articles/multifactor-authentication/api/otp.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Associate an OTP Authenticator
+description: How to associate an OTP authenticator
+---
+# Associate an OTP Authenticator
+
+In this tutorial, we will show you how you can configure your Auth0 tenant to allow the self-association of one-time password (OTP) authenticators.
+
+## Step 1. Trigger the MFA Error and Use the MFA Token to Associate the New Authenticator
+
+Whenever a user begins the authorization process and they do not have an active authenticator associated with their account, they will trigger the following MFA response when calling the `/oauth/token` endpoint:
+
+```json
+{
+  "error": "mfa_required",
+  "error_description": "Multifactor authentication required",
+  "mfa_token": "Fe26...Ha"
+}
+```
+
+You will use the MFA token instead of the standard access token to request association of a new authenticator.
+
+## Step 2: Use the Access Token to Request Association of the Authenticator
+
+Now that you have the appropriate access token, you can send the appropriate `POST` request to the `/mfa/associate` endpoint to request Association of your authenticator.
+
+To associate an authenticator where the challenge type is an SMS message containing a code that the user is then required to provide, make the following `POST` call to the `/mfa/associate` endpoint. Be sure to replace the placeholder values in the payload body shown below as appropriate.
```

Comment:
`To associate an authenticator where the challenge type is a TOTP code that the user is`